canonical/store-components
# Add optional links to be displayed in the third column with a top border

## Done
Add optional links to be displayed in the third column with a top border 

## QA
-Go to https://store-components-22.demos.haus/?path=/story/footer--default-story
-Check if there are optional links in the third column with a top border

## Fixes
Fixes: https://warthogs.atlassian.net/browse/WD-1616
